.changeset/swift-foxes-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@medusajs/medusa": patch
+---
+
+feat(medusa): allow registering overrides for fields, relations & validators easily
```

Comment:
todo: remove the validators here as there is nothing around the validator. Unless you want to add it before we merge?